syft/pkg/cataloger/ruby/parse_gemspec.go

```diff
@@ -36,14 +35,14 @@ var patterns = map[string]*regexp.Regexp{
 	"homepage": regexp.MustCompile(`.*\.homepage\s*=\s*["']{1}(?P<homepage>.*)["']{1} *`),
 
 	// match example:       files = ["exe/bundle".freeze, "exe/bundler".freeze]    --->    "exe/bundle".freeze, "exe/bundler".freeze
-	"files": regexp.MustCompile(`.*\.files\s*=\s*\[(?P<files>.*)\] *`),
+	"files": regexp.MustCompile(`.*\.files\s*=\s*\[(?P<files>.*)] *`),
```

Comment:
Why did we have to remove matching on the `\` here?

Reply:
IDE flagged this as a redundant escape (which appears correct)